### ISSUE TITLE
Serilog.Tests.Support nuget package

### ISSUE THREAD

**Issue by global-roam** [serilog-trace-listener/SerilogTraceListener#8]
the SerilogTraceListener.Tests project depends on a nuget package called "Serilog.Tests.Support" but this is not on nuget.org.  where Can I find it or can preferably can it be published to nuget.org?

**Comment by global-roam** [comment]
Found the source @ https://github.com/serilog-trace-listener/serilog-tests-support, still need the nuget package on nuget.org though.

**Comment by vossad01** [comment]
I am not near a capable device to look at it at the moment. At least
previously it was published to nuget as a private package.

**Comment by global-roam** [comment]
ok, should it be a private package though?  I guess the crux of the issue is that I cloned the repository and I can't build out of the box. I've worked around the issue because it was blocking me by pulling the related repository and referencing the project directly instead of the nuget package. but this means I have to jump through a few hoops to push my changes back and create a pull request to fix a bug.
